l10n/en/mozorg/home-m24.ftl

```diff
@@ -0,0 +1,106 @@
+m24-home-were-proudly-nonprofit = We’re proudly non-profit. Will you donate to { -brand-name-mozilla } today?
+m24-home-donate = Donate
+
+## AI gallery
+
+m24-home-tag-product = Product
+m24-home-tag-program = Program
+m24-home-tag-event = Event
+m24-home-read-more = Read more
+m24-home-get-started = Get started
+m24-home-join-the-movement = Join the movement:<br> AI for the people
+m24-home-our-mission-is = Our mission is to make it easy for people to build with, and collaborate on, open-source, trustworthy AI.
+m24-home-explore-mozilla-ai = Explore { -brand-name-mozilla-ai }
+m24-home-llamafile = { -brand-name-llamafile }
+m24-home-run-large-language = Run large language models (LLMs) easily on your computer with { -brand-name-llamafile } — no installation needed, and your data stays safe on your device.
+m24-home-mozilla-builders= { -brand-name-mozilla } Builders
+m24-home-turn-your-ai = Turn your AI idea into reality with our 12-week accelerator. Get funding, mentorship and a chance to pitch your innovation to investors.
+m24-home-lumigator = { -brand-name-lumigator}
+m24-home-find-the-just = Find the “just right” LLM for your needs, use case and data — like Goldilocks, but with sharper teeth.
+m24-home-can-ai-be = Can AI be trustworthy?
+m24-home-in-a-world = In a world where AI innovation is driven by a select few, we’re risking monopolization of this technology. Applying open-source concepts to AI could change that.
+m24-home-watch-now = Watch now
+m24-home-mozilla-ventures = { -brand-name-mozilla-ventures }
+m24-home-got-an-earlystage = Got an early-stage startup? Pitch your company to { -brand-name-mozilla-ventures } and secure funding to drive positive change for the future of AI and the internet.
+m24-home-common-voice = { -brand-name-common-voice }
+m24-home-donate-your-voice-to-make = Donate your voice to make speech technology more inclusive and accessible for everyone.
+m24-home-donate-your-voice = Donate your voice
+
+## Media Springboard
+
+m24-home-type = Type
+m24-home-author = Author
+m24-home-topic = Topic
+m24-home-intro = Intro
+m24-home-tag-article = Article
+m24-home-tag-podcast = Podcast
+m24-home-tag-video = Video
+m24-home-topic-news = News
+m24-home-topic-ai = Artificial Intelligence
+m24-home-topic-ps = Privacy & Security
+m24-home-headline-you-ai = Headline: You, AI and the internet — what’s really going on?
+m24-home-introducing-anonym = Introducing { -brand-name-anonym }: Raising the bar for privacy-preserving digital advertising.
+m24-home-keeping-genai-technologies = Keeping GenAI technologies secure is a shared responsibility.
+m24-home-from-hollywood-to = From Hollywood to hip hop, artists are negotiating AI consent.
+m24-home-mozilla-celebrates-groundbreaking = { -brand-name-mozilla } celebrates groundbreaking creators in new docuseries “{ -brand-name-firefox } Presents.”
```

Comment:
**Suggestion (non-blocking):** The term `docuseries` is common online today, but I'm not sure if it's recognized widely as a real word? (my spell checker doesn't seem to recognize it at least). 

Is it worth add an L10n comment to say it's short for "documentary series", or just using the two words for a simplified English string?

Reply:
✅